packages/cli/src/commands/generate-abi.ts

```diff
@@ -0,0 +1,66 @@
+import type { CommandModule, InferredOptionTypes } from "yargs";
+import { Hex, createWalletClient, http } from "viem";
+import { getSystems } from "../deploy/getSystems";
+import { getWorldDeploy } from "../deploy/getWorldDeploy";
```

Comment:
we should consider "lifting" these out of `deploy` dir if we're sharing them across different commands/utils

Reply:
Yup, and I think having separate directories for each command makes less sense with these new tools. We'll basically be reusing everything except for the `ensureX` functions that actually deploy the contracts